test/rulePriorities.js

```diff
@@ -71,53 +71,51 @@ describe('Rule Priorities', () => {
-        // take priority over tracker blocking and the tracker allowlist, but
+        // take priority over tracker blocking, the tracker allowlist and smarter encryption, but
         // not other features/allowlists.
+        assert.ok(AD_ATTRIBUTION_POLICY_PRIORITY > SMARTER_ENCRYPTION_PRIORITY)
         assert.equal(AD_ATTRIBUTION_POLICY_PRIORITY, CONTENT_BLOCKING_ALLOWLIST_PRIORITY)
         assert.ok(AD_ATTRIBUTION_POLICY_PRIORITY > TRACKER_BLOCKING_CEILING_PRIORITY)
         assert.ok(AD_ATTRIBUTION_POLICY_PRIORITY > TRACKER_ALLOWLIST_CEILING_PRIORITY)
         assert.ok(AD_ATTRIBUTION_POLICY_PRIORITY < GPC_HEADER_PRIORITY)
         assert.ok(AD_ATTRIBUTION_POLICY_PRIORITY < TRACKING_PARAM_PRIORITY)
-        assert.ok(AD_ATTRIBUTION_POLICY_PRIORITY < SMARTER_ENCRYPTION_PRIORITY)
         assert.ok(AD_ATTRIBUTION_POLICY_PRIORITY < UNPROTECTED_TEMPORARY_ALLOWLIST_PRIORITY)
         assert.ok(AD_ATTRIBUTION_POLICY_PRIORITY < SERVICE_WORKER_INITIATED_ALLOWING_PRIORITY)
         assert.ok(AD_ATTRIBUTION_POLICY_PRIORITY < USER_ALLOWLISTED_PRIORITY)
 
         // Tracking parameter protection, AMP protection  and GPC should take
-        // priority overtracker blocking, the tracker allowlist and the
+        // priority overtracker blocking, the tracker allowlist, smarter encryption and the
         // contentBlocking allowlist, but not over the other features/allowlists.
+        assert.ok(TRACKING_PARAM_PRIORITY > SMARTER_ENCRYPTION_PRIORITY)
         assert.equal(TRACKING_PARAM_PRIORITY, AMP_PROTECTION_PRIORITY)
         assert.equal(TRACKING_PARAM_PRIORITY, GPC_HEADER_PRIORITY)
         assert.ok(TRACKING_PARAM_PRIORITY > TRACKER_BLOCKING_CEILING_PRIORITY)
         assert.ok(TRACKING_PARAM_PRIORITY > TRACKER_ALLOWLIST_CEILING_PRIORITY)
         assert.ok(TRACKING_PARAM_PRIORITY > CONTENT_BLOCKING_ALLOWLIST_PRIORITY)
-        assert.ok(TRACKING_PARAM_PRIORITY < SMARTER_ENCRYPTION_PRIORITY)
         assert.ok(TRACKING_PARAM_PRIORITY < UNPROTECTED_TEMPORARY_ALLOWLIST_PRIORITY)
         assert.ok(TRACKING_PARAM_PRIORITY < SERVICE_WORKER_INITIATED_ALLOWING_PRIORITY)
         assert.ok(TRACKING_PARAM_PRIORITY < USER_ALLOWLISTED_PRIORITY)
 
-        // Smarter encryption should take priority over most features and
-        // allowlists, except for the unprotectedTemporary allowlist, the
-        // exception for ServiceWorker initiated requests and user allowlisting.
-        assert.ok(SMARTER_ENCRYPTION_PRIORITY > TRACKER_BLOCKING_CEILING_PRIORITY)
-        assert.ok(SMARTER_ENCRYPTION_PRIORITY > TRACKER_ALLOWLIST_CEILING_PRIORITY)
-        assert.ok(SMARTER_ENCRYPTION_PRIORITY > AD_ATTRIBUTION_POLICY_PRIORITY)
-        assert.ok(SMARTER_ENCRYPTION_PRIORITY > CONTENT_BLOCKING_ALLOWLIST_PRIORITY)
-        assert.ok(SMARTER_ENCRYPTION_PRIORITY > GPC_HEADER_PRIORITY)
-        assert.ok(SMARTER_ENCRYPTION_PRIORITY > TRACKING_PARAM_PRIORITY)
+        // Smarter encryption is lower priority than all other rules.
```

Comment:
Could you move this stanza to be above the "// Tracker allowlist should take priority over tracker blocking and smarter encryption..." one? The stanzas are ascending order of priority too.